specification/contosowidgetmanager/Contoso.Management/tspconfig.yaml

```diff
@@ -33,9 +33,9 @@ options:
     package-details:
       name: "@azure/arm-contoso"
   "@azure-tools/typespec-go":
-    service-dir: "sdk/resourcemanager/contoso"
-    package-dir: "armcontoso"
-    module: "github.com/Azure/azure-sdk-for-go/{service-dir}/{package-dir}"
+    service-dir: "{output-dir}/sdk/resourcemanager/contoso"
+    emitter-output-dir: "{service-dir}/armcontoso"
+    module: "github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/contoso/armcontoso"
```

Comment:
Can 'module' reference 'emitter-output-dir' to avoid same definition in two places, like: `github.com/Azure/azure-sdk-for-go/{emitter-output-dir}`

Reply:
I assume `emitter-output-dir` must be an absolute path?